### ISSUE TITLE
GPU memory keep increase after each epoch?

### ISSUE THREAD

**Issue by trangtv57** [facebookresearch/wav2letter#664]
I train my dataset with streaming_convnets. I set my batchsize = 8 when start and gpu mem: 4763MiB/
8116MiB ( my gpu is: 1080 Ti), and after 20k step its increase to approxi 5200 MiB and then at 40k step is 6699 MiB. 
The memory of gpu keep increase not decrease anytime, but It does not increase steadily, its just increase at a fix interval.
I have encountered this error before when training my custom model in pytorch, and i just simple using del loss, and del ouput at each iter. But I don't know how to do this in wav2letter? Can you help me. 
thank you.

**Comment by trangtv57** [comment]
sry, i think i have wrong setting, everything may be fine. so i will close this issue
